docs/Quick_start_guide.rst

```diff
@@ -0,0 +1,144 @@
+===========================
+Quick start guide
+===========================
+
+Usage
+-----
+
+Basic examples
+..............
+
+Here is a simple python script that reads an image, applies a median image filter (radius of 2 pixels), and writes the resulting image in a file.
+
+  ::
+
+    import itk
+
+    image = itk.imread("/path/to/my/image.png")
+    median = itk.MedianImageFilter.New(image, Radius = 2)
+    itk.imwrite(median, "path/to/my/output_image.png")
+
+There are two ways to instantiate filters with ITKPython:
+
+- Implicit (recommended): ITK type information is automatically detected from the data. Under the hood typed filter objects and images are created. 
```

Comment:
`Under the hood typed filter objects and images are created. `
-> `Typed filter objects and images are implicitly created. `